modules/fiftyone_degrees/device_detection/account_info_extractor.go

```diff
@@ -0,0 +1,39 @@
+package device_detection
+
+import (
+	"github.com/tidwall/gjson"
+)
+
+type AccountInfo struct {
+	Id string
+}
+
+type AccountInfoExtractor struct{}
+
+func NewAccountInfoExtractor() *AccountInfoExtractor {
+	return &AccountInfoExtractor{}
+}
+
+// Extract extracts the account information from the payload
+// The account information is extracted from the publisher id or site publisher id
+func (x AccountInfoExtractor) Extract(payload []byte) *AccountInfo {
```

Comment:
As a general comment applicable to all public functions and variables, please assess whether these need to be exported. I don't think most of these are meant to be accessed outside of the `device_detection` package, in which case they should begin with a lower case letter.

Reply:
totally agree, thanks